### PR TITLE
Rely on `.gnarrc` for dependency management, add `rails` command

### PR DESCRIPTION
**Pull request by alxjrvs**
<Better Description incoming>

TL;DR: 
 - relies on `.gnarrc` to do dependency management 
 - add `rails new` command  with passthrough options via `--`
 
 Ideally this would be two PR's, but I stupidly chose to fork and it got very silly.

### CODE REVIEW

File: src/commands/eslint/index.ts [hunk 1]
```diff
@@ -1,127 +1,20 @@
-import { writeFileSync } from 'node:fs'
+import { Gnarrc } from '../../utils/gnarrc'
 import { PackageInstaller } from '../../utils/package-installer'
 import PackageJson from '../../utils/package-json'
-import { Prettier } from '../prettier'
```
Comment (efertsch) [inline src/commands/eslint/index.ts:4]: 😍  look at all this redddddd

**Review by efertsch — COMMENTED** [review]
Not seeing any red flags here but I also felt pretty out of my depth reading this over 😅 Getting a second set of eyes (or two) wouldn't hurt! cc @mikestone14 & @ngmaloney 

On another note, this is really impressive work, Alex. Thanks so much for taking the time to go through all of this and to get it running. I think this has the potential to greatly improve our workflow/internal processes at The Gnar ❤️

File: src/commands/eslint/index.ts (hunk 1 above)
Comment (alxjrvs) [reply src/commands/eslint/index.ts:4]: Right???? lol'

**Comment by alxjrvs** [comment]
This got really dumb to manage as a feature branch from a fork, so I closed it down and created #31, which is sourced from this repo.

**Pull request closed without merging**